project/scripts/get_genre_data.py

```diff
@@ -0,0 +1,40 @@
+# To execute: $ python manage.py runscript get_genre_data (optional: --script-args argument_name) or
+# $ python manage.py runscript get_genre_data --script-args first_time=True
+
+
+import requests
+from pprint import pprint #for debugging
+from django.conf import settings
+from movieswyper.models import Movie, Genre
+
+def run(first_time=False):
+    if first_time:
+        run_first_time()
+    else:
+        # TODO: Update function
+        pass
+
+def run_first_time():
+    url = f"https://api.themoviedb.org/3/genre/movie/list?api_key={settings.TMDB_API_KEY}&language=en-US"
+    response = requests.get(url)
```

Comment:
URL parameters have special escaping rules. Your current API key might happen to work unescaped, but this approach doesn't work in general. Instead, you should pass the GET parameters to the `params` argument of `requests.get`, so that `requests` can do the proper escaping of these values and append them to the URL:
```suggestion
    url = f"https://api.themoviedb.org/3/genre/movie/list'
    response = requests.get(url, params={
        'api_key': settings.TMDB_API_KEY,
        'language': 'en-US',
    })
```